recipes/quantlib/LICENSE.TXT

```diff
@@ -0,0 +1,199 @@
+QuantLib is
```

Comment:
I see the file is also in the repository. So you should be able to just use that file instead of adding it again here.